### DISCUSSION TITLE
Is Threatmapper Docker Agent supporting Unraid Server?

### DISCUSSION THREAD

**Discussion by nekromantik** [deepfence/ThreatMapper#2396] [General]
So I got a unraid server which is FreeBSD based server and its running docker.
I installed the docker container agent and it shows info about containers but no scans work.
I cant even get diagnostic logs as its empty.

**Comment by noboruma** [comment]
Hello @nekromantik, FreeBSD is not supported at the moment, we only support GNU/Linux at the moment. Any help to support FreeBSD is welcomed!

**Reply by nekromantik** [reply]
Hi So docker running on FreeBDS wont work then? My containers are all alpine linux based.

**Reply by noboruma** [reply]
@nekromantik This is correct, we utilize kernel features under the hood and interacts closely with it, so even GNU/linux based container will not work properly on a FreeBSD kernel.
Internally we don't have any plan (yet?) to support FreeBSD. There are some serious limitations like the lack of eBPF support. 
With that being said, any PR that proposes some minimal support for FreeBSD will be reviewed and taken seriously.
